samcli/lib/delete/cfn_utils.py

```diff
@@ -116,13 +116,16 @@ def wait_for_delete(self, stack_name):
 
         # Wait for Delete to Finish
         waiter = self._client.get_waiter("stack_delete_complete")
-        # Poll every 5 seconds.
-        waiter_config = {"Delay": 30}
+        # Poll every 5 seconds and set max attempts to be 3.
+        waiter_config = {"Delay": 5, "MaxAttempts": 3}
```

Comment:
Quick question: with max tries at 3 and delay at 5 seconds, is it safe to assume that delete operations will take at most 15 seconds?

Reply:
This is just for polling, it could take longer, but we poll for status every 5 seconds. 30 seconds seemed long to me.